howso/engine/trainee.py

```diff
@@ -3170,9 +3191,11 @@ def react_group(
         if isinstance(self.client, HowsoPandasClientMixin):
```

Comment:
```suggestion
        if isinstance(self.client, AbstractHowsoClient):
```
(Mixin no longer overrides this method)